dev_tools/commitizen/CHANGELOG.md.j2

```diff
@@ -0,0 +1,23 @@
+{# Jinja2 changelog template used by Commitizen #}
+{% for entry in tree %}
+
+## {{ entry.version }}{% if entry.date %} ({{ entry.date }}){% endif %}
+
+{# Sole difference from default template is link to the detailed GitHub release notes #}
```

Comment:
Consider updating this comment to simplify and avoid the specificity which may otherwise change in the future (i.e. incur a developer having to remember to update this comment if things change).
```suggestion
{# Add link to the detailed GitHub release notes #}
```